src/app/clusters/energy-evse-server/EnergyEvseTestEventTriggerDelegate.h

```diff
@@ -0,0 +1,66 @@
+/*
+ *
+ *    Copyright (c) 2023 Project CHIP Authors
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+#pragma once
+
+#include <app-common/zap-generated/cluster-objects.h>
+#include <app/TestEventTriggerDelegate.h>
+
+namespace chip {
+
+enum class EnergyEvseTrigger : uint64_t
```

Comment:
Should probably document how this corresponds to the delegate's `enableKey`.  I assume these 8-byte values are possible enable key values, but endianness needs to be documented or something....

Alternately, the constructor could take these values directly and do the endianness bits itself as needed, if that's the intent?

Reply:
Ok - I was just following the same pattern as src/app/clusters/smoke-co-alarm-server/SmokeCOTestEventTriggerDelegate.h

These are the codes use to specific a specific type of test event trigger, not the enableKey.

Reply:
Added comment

Reply:
Thank you, that makes a lot more sense.